module/VuFind/src/VuFind/ILS/Driver/Virtua.php

```diff
@@ -1216,23 +1216,24 @@ public function getMyProfile($patron)
         }
         [$last_name, $first_name] = $this->getLastAndFirstName($result['NAME']);
         $split = strpos($first_name, ' ');
+        // TODO: Is this necessary
```

Comment:
The purpose appears to be to strip off middle names. Probably not necessary, though this PR is probably not the best place to change it. (I suspect no one is using this driver in its current form in any case -- the only Virtua library I'm aware of is National Library of Ireland, and they use a heavily customized local driver).